### PR TITLE
Rename `src` to `jiant`

### PR DESCRIPTION
**Pull request by W4ngatang**
Rename `src` to jiant, and accordingly do the following:

- switch import from `src` in `main.py` to `jiant`

- remove relative imports that go outside the current directory to absolute imports using `jiant`.

### CODE REVIEW

**Comment by zphang** [comment]
What is the rationale for: 

>remove relative imports that go outside the current directory to absolute imports using jiant.

File: jiant/utils/config.py [hunk 1]
```diff
@@ -1,17 +1,18 @@
...
+import json
 import types
+import random
+import pyhocon
+import argparse
 import logging as log
+
 log.basicConfig(format='%(asctime)s: %(message)s',
                 datefmt='%m/%d %I:%M:%S %p', level=log.INFO)
+from . import hocon_writer
```
Comment (iftenney) [inline jiant/utils/config.py]: should this be `from jiant.utils import hocon_writer` ?

Comment (W4ngatang) [reply jiant/utils/config.py]: I didn't change relative imports that were really deep in the repo and importing from the same directory. You think it's better to switch everything to absolute imports?

**Comment by W4ngatang** [comment]
> 
> 
> What is the rationale for:
> 
> > remove relative imports that go outside the current directory to absolute imports using jiant.

@zphang mainly to avoid having really long import lines when it seems likely that those files will stay in the same folder. Do you think it's better to stick to absolute imports always?

**Comment by sleepinyourhat** [comment]
As soon as you merge this, make sure to fix all mentions in .sh scripts, readmes, the site, and the tutorials.

File: jiant/utils/config.py (hunk 1 above)
Comment (iftenney) [reply jiant/utils/config.py]: Yes, for consistency and less brittle if we move anything.

**Comment by sleepinyourhat** [comment]
And fix broken tests. Looks like this probably won't make it into 0.9, though if you're dead sure everything works and you fix the tests, feel free to merge.

**Comment by sleepinyourhat** [comment]
LGTM

**Comment by iftenney** [comment]
Can you fix the `probing/src` symlink so that the scripts in that directory will still work? (and anywhere else something similar is used)

**Comment by iftenney** [comment]
(otherwise, LGTM)

**Pull request merged**